submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/wiremock/FireMockDatabase.kt

```diff
@@ -8,11 +8,12 @@ import uk.ac.ebi.fire.client.model.FileSystemEntry
 import uk.ac.ebi.fire.client.model.FireApiFile
 import java.io.File
 import java.time.Instant
+import java.util.Collections
 
 class FireMockDatabase(
     private val fileSystem: FireMockFileSystem,
 ) {
-    private val recordsById: MutableMap<String, DbRecord> = mutableMapOf()
+    private val recordsById: MutableMap<String, DbRecord> = Collections.synchronizedMap(mutableMapOf())
```

Comment:
I think we should use ConcurrentHashMap check https://www.geeksforgeeks.org/difference-between-concurrenthashmap-and-synchronizedhashmap/